### PR TITLE
[MSUTB] Add Portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
Add new Portuguese (pt-PT) translation.

### CODE REVIEW

File: dll/win32/msutb/lang/pt-PT.rc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+IDD_CLOSELANGBAR DIALOGEX 0, 0, 200, 105
+CAPTION "Barra de idiomas"
+STYLE WS_POPUPWINDOW | WS_CAPTION | DS_MODALFRAME
+FONT 8, "MS Shell Dlg"
+{
+    LTEXT "Fechou a barra de idiomas.\r\n\r\nPara utilizar a barra de idiomas posteriormente, marque ""Mostrar a barra de idiomas na área de trabalho"" na caixa de diálogo Configurações da barra de idiomas, no painel de controle Opções regionais e de idioma.", -1, 5, 5, 190, 60
+    AUTOCHECKBOX "&Não mostrar esta mensagem novamente.", IDC_CLOSELANGBAR_CHECK, 5, 70, 145, 10
+    DEFPUSHBUTTON "&OK", IDOK, 80, 85, 55, 15
+    PUSHBUTTON "&Cancelar", IDCANCEL, 140, 85, 55, 15
```
Comment (JoachimHenze) [inline dll/win32/msutb/lang/pt-PT.rc:56]: Please prove with a screenshot from portugese 2k3sp2 that Windows does use an accelerator here for OK and Cancelar. You did that in multiple dialogs, and I do somehow have my doubts that it is correct, based on findings in other translations.

Comment (zecarlos1957) [reply dll/win32/msutb/lang/pt-PT.rc]: You're right. There are no accelerators on the Ok and Cancel buttons, at least in WinXP. Thanks. Done.

File: dll/win32/msutb/lang/pt-PT.rc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2023-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&OK"
+    IDS_CANCEL           "&Cancelar"
```
Comment (HBelusca) [inline dll/win32/msutb/lang/pt-PT.rc:13]: Normally, OK and Cancel shouldn't require an explicit key accelerator (they would default with ENTER and ESC keys respectively). English lang don't have the accelerator, and I know @JoachimHenze started to remove such redundant ones in shell32 lang files.

File: dll/win32/msutb/lang/pt-PT.rc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    IDS_IGNORE           "&Ignorar"
+    IDS_YES              "&Sim"
+    IDS_NO               "&Não"
+
+    IDS_SETTINGS         "&Definições..."
+    IDS_RESTORELANGBAR   "Restaurar a barra de idiomas"
+    IDS_LANGUAGE         "Linguagem"
+    IDS_LANGUAGEBUTTON   "Botão Linguagem"
+    IDS_TRANSPARENCY     "&Transparência"
+    IDS_TEXTLABELS       "T&exto da etiqueta"
```
Comment (HBelusca) [inline dll/win32/msutb/lang/pt-PT.rc:25]: OK here -- @JoachimHenze Observe however that the english resource seems to have a duplicated `&t` accelerator in that text, with respect to the previous (Transparency) one; my suggestion would be to either put the acccel on the 'e' of "Text Labels", or, to put it on the "L" of Labels instead.

File: dll/win32/msutb/lang/pt-PT.rc [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    IDS_MINIMIZE         "&Minimizar"
+    IDS_CLOSELANGBAR     "&Fechar a barra de idiomas"
+    IDS_EXTRAICONS       "&Ícones adicionais na barra de tarefas"
```
Comment (HBelusca) [inline dll/win32/msutb/lang/pt-PT.rc:28]: @zecarlos1957 Is accentued I a good keyboard accelerator? wouldn't this require the user to press more than two keys (the accent, the capital I, and Alt)?

File: dll/win32/msutb/lang/pt-PT.rc [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    IDS_ADJUSTLANGBAND   "&Ajuste a posição da banda de idioma"
+    IDS_LEFTCLICK        "Clique esquerdo"
+END
+
+IDD_CLOSELANGBARNOBAND DIALOGEX 0, 0, 200, 105
+CAPTION "Barra de idiomas"
+STYLE WS_POPUPWINDOW | WS_CAPTION | DS_MODALFRAME
+FONT 8, "MS Shell Dlg"
+{
+    LTEXT "Fechou a barra de idiomas.\r\n\r\nPara utilizar a barra de idiomas posteriormente, marque ""Mostrar a barra de idiomas na área de trabalho"" na caixa de diálogo Configurações da barra de idiomas, no painel de controle dos Serviços de Texto.", -1, 5, 5, 190, 60
```
Comment (HBelusca) [inline dll/win32/msutb/lang/pt-PT.rc:42]: Cc @katahiromz : I don't think that we have to use `\r\n` escapes for newlines in resources for dialogs, just `\n` should be enough.

File: dll/win32/msutb/lang/pt-PT.rc (hunk 3 above)
Comment (JoachimHenze) [reply dll/win32/msutb/lang/pt-PT.rc:25]: An owner of a pt-PT-Windows should decide that.

File: dll/win32/msutb/lang/pt-PT.rc (hunk 2 above)
Comment (zecarlos1957) [reply dll/win32/msutb/lang/pt-PT.rc]: I will keep that in mind. Thanks.

File: dll/win32/msutb/lang/pt-PT.rc (hunk 5 above)
Comment (katahiromz) [reply dll/win32/msutb/lang/pt-PT.rc]: You can test `"test\n\ntest"` with Resource Hacker.

Comment (HBelusca) [reply dll/win32/msutb/lang/pt-PT.rc]: Note that these \r\n are also in the english translation, they would need to be replaced by \n in a separate PR/commit (unrelated to the present PR though.)

**Review by HBelusca — APPROVED** [review]

**Pull request merged**